src/kartograf/mapping_metrics/metric_shape_difference.py

```diff
@@ -125,14 +142,18 @@ def get_rdmol_shape_distance(self, molA:Chem.Mol, molB: Chem.Mol)->float:
         float
             raw result of the applied rdkit shape distance.
         """
-        if(any([x is None for x in [molA, molB]])):
-            if(all([x is None for x in [molA, molB]])):
+        if any([x is None for x in [molA, molB]]):
```

Comment:
minor, but don't put square brackets inside an `any()` statement.  `any(x is None for x in [molA, molB)` can short circuit if molA is None, but if you put the square brackets the entire list must be evaluated.

e.g. in C:
```c
any = False
for (val: vals):
    if (val == None):
        any = True
return any
```
vs
```c
for (val: vals):
    if (val == None):
        return True
return False
```

Reply:
oh cool I did not know!